lib/rbac/filterer.rb

```diff
@@ -411,7 +431,7 @@ def include_references(scope, klass, include_for_find, exp_includes, skip)
 
       ref_includes = Hash(include_for_find).merge(Hash(exp_includes))
       unless polymorphic_include?(klass, ref_includes)
-        scope = scope.references(include_for_find).references(exp_includes)
+        scope = scope.references(self.class.includes_to_references(klass, include_for_find)).references(self.class.includes_to_references(klass, exp_includes))
```

Comment:
RE: my comment about where this should be live, it feels like it might make sense to call this like `klass.includes_to_references(include_for_find)`... so maybe a module added to AR.  I could be totally wrong.  😉